packages/core/useSafeArea/index.ts

```diff
@@ -0,0 +1,56 @@
+import { ref } from 'vue-demi'
+import { isClient, useCssVar, useDebounceFn, useEventListener } from '@vueuse/core'
+
+const topVarName = '--vue-use-safe-area-top'
```

Comment:
I think `vueuse` here should be a single word, so `--vueuse-safe-area-top` instead

Reply:
ok~